### ISSUE TITLE
Implement Cameron's signed invitation proposal

### ISSUE THREAD

**Issue by mickvandijke** [torrust/torrust-index-archive#85]
From: https://github.com/torrust/torrust-index/issues/30#issuecomment-1140497385

- [ ] https://github.com/torrust/torrust-index-backend/issues/71
- [ ] Add signup form field for invitation.

**Comment by da2ce7** [comment]
I think that the signed invitation should be another TEP. It would depend on [TEP 12](https://github.com/torrust/teps/pull/9).

**Comment by da2ce7** [comment]
> I think that the signed invitation should be another TEP. It would depend on [TEP 12](https://github.com/torrust/teps/pull/9).

I have been working on this, but haven't published a draft yet.

**Comment by mickvandijke** [comment]
Hey @da2ce7 ,

I started working on this, but then I realized that users can only create an invite if they have a public key coupled to their account. This would be always be the case if we implement TEP 12 of course.

One question that still sits in my mind though is where does a user go to generate an invite? Since users need to be able to generate a key while offline. We can't make a tool on the index, since that would require an internet connection to reach. Did you have something in mind for this?
